addons/greatsql/values.yaml

```diff
@@ -0,0 +1,37 @@
+image:
+  registry: docker.io
+  repository: greatsql/greatsql
+  pullPolicy: IfNotPresent
+  tag: 8.0.32-25
+  prom:
+    repository: prom
+    pullPolicy: IfNotPresent
+    mysqld_exporter:
+      name: mysqld-exporter
+      tag: v0.14.0
+
+## MySQL Authentication parameters
+auth:
+  ## MYSQL_ROOT_HOST
+  rootHost: "%"
+  ## @param auth.createDatabase Whether to create the .Values.auth.database or not
+  ##
+  createDatabase: true
```

Comment:
seems values from line 19 to line 32 are not used?

Reply:
Yes, it's for the next version, and I've removed it